app/config/prompts/summaries_model_prompt_Phi-3.5-instruct.txt

```diff
@@ -6,8 +6,8 @@ attributes. Prioritize essential data for efficient storage
 and retrieval, and omit any unnecessary details.
 <|end|>
 <|user|>
-This is the resource:\n
----------------------\n
-{query}\n
----------------------\n
+This is the resource:
+---------------------
```

Comment:
Is there a point in having these lines when you later replace all line breaks (\n)?

Reply:
I will remove them, I thought I had removed all of them